pybossa/view/projects.py

```diff
@@ -1698,6 +1700,106 @@ def bulk_priority_update(short_name):
     except Exception as e:
         return ErrorStatus().format_exception(e, 'priorityupdate', 'POST')
 
+@crossdomain(origin='*', headers=cors_headers)
+@blueprint.route('/<short_name>/tasks/assign-workersupdate', methods=['GET', 'POST'])
+@login_required
+@admin_or_subadmin_required
+def bulk_update_assign_worker(short_name):
+   
+    response = {}
+    project, owner, ps = project_by_shortname(short_name)
+    data = json.loads(request.data)
+
+    if data.get("add") is None and data.get("remove") is None:
+        # read data and return users
+
+        task_id = data.get("taskId")
+        bulk_update = False
+        assign_user_emails = []
+        if task_id:
+            # use filters tp populate user list
+            t = task_repo.get_task_by(project_id=project.id,
+                                        id=int(task_id))
+            assign_user_email = []
+            if t.user_pref is not None and isinstance(t.user_pref, dict):
+                assign_user_emails = set(t.user_pref.get("assign_user", []))
+        else:
+            bulk_update = True
+            args = parse_tasks_browse_args(json.loads(data.get('filters', 'null')))
```

Comment:
This line could possible use a check after it to see if the 'filters' parameter exists, otherwise an error may be thrown.

For example, this may be a fix:

```python
filters = json.loads(data.get('filters', 'null')) or dict()
args = parse_tasks_browse_args(filters)
```